cypress/integration/dib-travel/company-employees/groups.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { Groups } from '../../../models/company-employees/groups';
+
+describe('Company Employees - Groups Page', () => {
+  let groupsDetails: Groups;
+
+  before(() => {
+    cy.fixture('company-employees/groups').then((group) => {
+      groupsDetails = group;
+    });
+  });
+
+  it('should allow regular user to add new group', () => {
+    cy.login();
+    cy.get('dib-layout dib-hamburger-icon').click();
+    cy.get('[routerLink="/people-management/groups"]').click();
+    cy.get('dib-people-management dib-groups ui-button button').contains('Add Group').click();
+    cy.get('.cdk-overlay-container dib-group-dialog input[placeholder="group name*"]').type(groupsDetails.name);
+    cy.get('.cdk-overlay-container dib-group-dialog dib-assign-members .members mat-checkbox').click();
```

Comment:
Please try to find selector without Angular Material classes (`mat-*`)

Reply:
I changed mat-* selector to .mat-checkbox-layout class